cylc/flow/network/scan.py

```diff
@@ -545,14 +545,12 @@ def _callback(_, entry):
 
     # NOTE: use the public DB for reading
     # (only the scheduler process/thread should access the private database)
-    db_file = Path(get_workflow_run_dir(flow['name'], 'log', 'db'))
+    db_file = Path(get_workflow_run_dir(
+        flow['name'], WorkflowFiles.LogDir.DIRNAME, WorkflowFiles.LogDir.DB
+    ))
     if db_file.exists():
-        dao = CylcWorkflowDAO(db_file, is_public=False)
```

Comment:
Huh, scan was reading private DBs in spite of the comment above? 😬

Reply:
It was until https://github.com/cylc/cylc-flow/commit/b338a91d5db0e1e60ca7ca16c3be28c86c630693